packages/react-components/react-table/src/hooks/useKeyboardResizing.ts

```diff
@@ -54,57 +54,85 @@ export function useKeyboardResizing(columnResizeState: ColumnResizeState) {
       case Enter:
       case Escape:
         stopEvent();
-        disableInteractiveMode();
+        // Just blur here, the onBlur handler will take care of the rest (disableInteractiveMode).
+        refMap.get(columnId)?.current?.blur();
```

Comment:
why an additional event listener? you have everything you need to focus the header button here.

Reply:
Not sure I follow, there is 1 keyboard handler for resizing and 1 blur handler for leaving this mode with esc, enter or space, or outside click. What is the additional event listerener? :)

Reply:
This talks about the same thing as https://github.com/microsoft/fluentui/pull/28493#discussion_r1266648171, gonna resolve this one and let's continue in the other thread